modules/heat_conduction/src/JouleHeatingHeatGeneratedAux.C

```diff
@@ -0,0 +1,41 @@
+//* This file is part of the MOOSE framework
```

Comment:
Please create an "auxkernels" directory under source and move this file into that folder.